Assets/Scripts/Models/Furniture.cs

```diff
@@ -183,7 +184,7 @@ protected Furniture(Furniture other)
     {
         this.objectType = other.objectType;
         this.Name = other.Name;
-        this.baseType = other.baseType;
+        this.typeTags = new HashSet<string>(other.typeTags);
```

Comment:
@henninglive the copy constructor is called here,  it copies everything inside other.typeTags into this.typeTags